syntax = "proto3";

// The greeting service definition.
service User {
    // Sends a greeting
    rpc Search (UserRequest) returns (UserReply) {
    }
}

message UserRequest {
    string search_term = 1;
}

message UserReply {
    string user_name = 1;
    string user_id = 2;
    string group_id = 3;
}